packages/graphql/src/schemaTypes/objectTypes/gql-ProjectPreferences.ts

```diff
@@ -7,9 +7,5 @@ export const ProjectPreferences = objectType({
     t.string('testingType', {
       description: 'The preferred testing type to start in',
     })
-
-    t.string('browserPath', {
-      description: 'The preferred browser to launch',
-    })
```

Comment:
Would be cool to have a ticket to add this back somewhere. The number of times I forgot to click to start a browser is non-negligeable.

Reply:
There is UNIFY-1059 and UNIFY-1243 though I think UNIFY-1243 may be a duplicate. Does that/do those cover this ask?

Reply:
UNIFY-1536